.github/workflows/ci.yml

```
@@ -166,15 +166,15 @@
     steps:
     - uses: actions/checkout@v4
     - name: jenkins-latest
-      run: docker build . --file ./jenkins/Dockerfile.latest --tag jenkins-latest:$(date +%s)
+      run: docker build . --file ./jenkins/latest.Dockerfile --tag jenkins-latest:$(date +%s)
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:72: Quote this to prevent word splitting

(shellcheck)